### ISSUE TITLE
Any disable api?

### ISSUE THREAD

**Issue by ido1wald** [arnthor3/react-bootstrap-toggle#28]
The disable props on the button is kinda buggy.
Any other solution?

**Comment by arnthor3** [comment]
What kind of a browser are you using? I admit I only test in Chrome and Firefox.
In what way is it buggy?

I think you could just return false from the onclick handler to at least not toggle the states.

If you give me a better description I can check this out tonight.

**Comment by arnthor3** [comment]
Ah, I understand. It's rendered with no dimensions and when the visibility is toggled it does not recalculate the width and height. 
I will add resize observer to the toggle as a prop.
```js
<Toggle 
  {...restOfProps}
  recalculateOnResize={true}
/>
```
That way you don't have to manually update the component.

**Comment by ido1wald** [comment]
That's really nice :)
I just added a style tag to the component to have a fixed size such as
```
<Toggle 
style = { {height: "100px", width: "100px" }} 
/>
```

Thanks for all the effort :)

**Comment by arnthor3** [comment]
Just released 2.3.0, it does include the resize prop. I am going to leave this open until I updated the tests accordingly. Thanks for the catch.
